network-api/networkapi/buyersguide/tests.py

```diff
@@ -317,3 +317,11 @@ def test_product_view_404(self):
         request = self.factory.get('/privacynotincluded/products/this is not a product')
         request.user = self.user
         self.assertRaises(Http404, product_view, request, 'this is not a product')
+
+    def test_category_view_404(self):
+        """
+        Test that the category view raises an Http404 if the category name doesn't exist
+        """
+        request = self.factory.get('/privacynotincluded/categories/this is not a category')
+        request.user = self.user
+        self.assertRaises(Http404, product_view, request, 'this is not a category')
```

Comment:
you should import the `category_view` from `networkapi.buyersguide.views` and pass it here instead of the product view.

Reply:
good catch